gio/src/io_extension_point.rs

```diff
@@ -0,0 +1,186 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use glib::translate::*;
+use glib::Type;
+use std::fmt;
+use std::ptr;
+
+use crate::io_extension::IOExtension;
+
+/// Builder for extension points.
+#[derive(Debug)]
+pub struct IOExtensionPointBuilder<'a> {
```

Comment:
```suggestion
#[must_use("Builder doesn't do anything unless built")]
pub struct IOExtensionPointBuilder<'a> {
```

Or something like that